packages/calendar/src/types.ts

```diff
@@ -212,6 +212,7 @@ export type TimeRangeSvgProps = Dimensions & { data: CalendarDatum[] } & Partial
                 role: string
                 weekdayLegendOffset: number
                 weekdayTicks: Array<0 | 1 | 2 | 3 | 4 | 5 | 6>
+                firstDayOfWeek: number
```

Comment:
I think it should be a union as we don't support any number.

Reply:
Maybe it would be even better to use `'monday' | 'tuesday' | ...`, we don't really know if `0` stands for Monday or Sunday, I feel it would provide a clearer API.

Reply:
Updated with enum.